IntegrationTests/TestSuites/Sources/PrimaryTests/main.swift

```diff
@@ -200,7 +200,9 @@ try test("Closure Lifetime") {
         }
         c1.release()
         // Call a released closure
+        print("======= BEGIN OF EXPECTED FATAL ERROR =====")
         _ = try expectThrow(try evalClosure.throws(c1))
```

Comment:
Should expectThrow contain these prints to avoid repetition?

Reply:
`expectThrow` is not for crash tests, so I added another helper function